Mathlib/GroupTheory/Perm/Support.lean

```diff
@@ -701,8 +701,7 @@ variable {α : Type*} [Fintype α] [DecidableEq α] {σ τ : Perm α}
 theorem support_conj : (σ * τ * σ⁻¹).support = τ.support.map σ.toEmbedding := by
   ext
   simp only [mem_map_equiv, Perm.coe_mul, Function.comp_apply, Ne, Perm.mem_support,
-    Equiv.eq_symm_apply]
```

Comment:
Just for my learning experience: this change is spelling out what was previously an implicit definitional equality?

Reply:
Yeah, this left a goal `··· σ⁻¹ ··· ↔ ··· Equiv.symm σ ···` which is solved more cleanly with the lemma that seys those are equal.